File: WebApi/Models/PaymentTypeDto.ts
export interface PaymentTypeDto {
  id: number;
  name: string;
  description?: string;
  createdDate: string;
  updatedDate?: string;
  isActive: boolean;
  createdBy?: string;
  updatedBy?: string;
  createdByFullUser?: string;
  updatedByFullUser?: string;
  deletedByFullUser?: string;
}

export interface PaymentTypeGetDto {
  id: number;
  name: string;
  description?: string;
  createdDate: string;
  updatedDate?: string;
  isActive: boolean;
  createdBy?: string;
  updatedBy?: string;
  createdByFullUser?: string;
  updatedByFullUser?: string;
  deletedByFullUser?: string;
}

export interface CreatePaymentTypeDto {
  name: string;
  description?: string;
}

export interface UpdatePaymentTypeDto {
  id: number;
  name: string;
  description?: string;
}

File: DTOs/QuotationDto/PaymentTypeDto.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace cms_webapi.DTOs
{
    public class PaymentTypeDto : BaseEntityDto
    {
        public string Name { get; set; } = string.Empty;
        public string? Description { get; set; }
        public bool IsActive { get; set; }
        public string? CreatedBy { get; set; }
        public string? UpdatedBy { get; set; }
    }

    public class CreatePaymentTypeDto
    {
        [Required]
        [MaxLength(100)]
        public string Name { get; set; } = string.Empty;

        [MaxLength(500)]
        public string? Description { get; set; }
    }

    public class UpdatePaymentTypeDto
    {
        [Required]
        public long Id { get; set; }

        [Required]
        [MaxLength(100)]
        public string Name { get; set; } = string.Empty;

        [MaxLength(500)]
        public string? Description { get; set; }
    }

    public class PaymentTypeGetDto : BaseEntityDto
    {
        public string Name { get; set; } = string.Empty;
        public string? Description { get; set; }
        public bool IsActive { get; set; }
        public string? CreatedBy { get; set; }
        public string? UpdatedBy { get; set; }
    }
}
